pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -1089,6 +1089,15 @@ func (l *LibvirtDomainManager) generateConverterContext(vmi *v1.VirtualMachineIn
 		c.GPUHostDevices = gpuHostDevices
 	}
 
+	if _, err := os.Stat("/dev/kvm"); errors.Is(err, os.ErrNotExist) {
+		if err != nil {
+			return nil, err
+		}
+		c.DevKvmExists = false
+	} else {
+		c.DevKvmExists = true
+	}
```

Comment:
```suggestion
	if _, err := os.Stat("/dev/kvm"); err != nil {
		if !errors.Is(err, os.ErrNotExist) {
			return nil, err
		}
		c.DevKvmExists = false
	} else {
		c.DevKvmExists = true
	}
```

Like this it makes more sense to me.

Reply:
thanks.